pkgs/development/python-modules/diagrams-as-code/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  diagrams,
+  pydantic,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "diagrams-as-code";
+  version = "0.0.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dmytrostriletskyi";
+    repo = "diagrams-as-code";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-cd602eQvNCUQuCdn/RpcfURcDHjXLZ0gAG+SObB++Q0=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    diagrams
+    pydantic
+    pyyaml
+  ];
+
+  pythonRelaxDeps = [
+    "diagrams"
+    "pydantic"
+  ];
+
+  pythonImportsCheck = [ "diagrams_as_code" ];
+
+  doCheck = false; # no tests
+
+  meta = {
+    description = "Declarative configurations using YAML for drawing cloud system architectures";
+    homepage = "https://github.com/dmytrostriletskyi/diagrams-as-code";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ sigmanificient ];
```

```suggestion
    maintainers = with lib.maintainers; [ sigmanificient ];
    mainProgram = "diagrams-as-code";
```
https://github.com/dmytrostriletskyi/diagrams-as-code/blob/f7682bbf8cd82ea250aad6b6779fcdde4372a7d8/setup.py#L42-L46

---

Moved it to `pkgs/by-name`